skytemple_randomizer/randomizer/npc.py

```diff
@@ -312,8 +334,3 @@ def _randomize_actors(self, string_file, pokemon_string_data) -> dict[int, int]:
             FileType.SIR0.serialize(FileType.SIR0.wrap_obj(actor_list)),
         )
         return mapped
-
-    @staticmethod
-    def _get_name(string_file: Str, index: int, pokemon_string_data):
```

Comment:
Removed this method in favour of the new `get_pokemon_name` method in `utils.py` - this allows Pokemon names to be read from more than just this file.